apps/condo/domains/marketplace/utils/testSchema/index.js

```diff
@@ -46,9 +48,38 @@ async function updateTestInvoiceContext (client, id, extraAttrs = {}) {
     return [obj, attrs]
 }
 
+async function createTestMarketCategory (client, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const attrs = {
+        dv: 1,
+        sender,
+        name: faker.random.alphaNumeric(8),
```

Comment:
is name `alphaNumeric`? :)

Reply:
it's a string, no matter what character it stores, it will still be a string

Reply:
I'll even do that))
<img width="647" alt="image" src="https://github.com/open-condo-software/condo/assets/64303474/6a6e11fc-2785-47ab-a16c-62f4ec54717b">

Reply:
> I'll even do that)) 
OMG)